### PR TITLE
[13.0][ENH]Purchase_Work_Acceptance

### PR DESCRIPTION
**Pull request by kranokporn**
Enhance the account_move_view by adding attributes required type.

cc. @ps-tubtim

### CODE REVIEW

**Comment by kittiu** [comment]
I think it is better to change the logic of compute field, so that require_wa = True for case in_invoice only. Also please make it multi.

Note: in v12 we don't have this problem, because we separate view.

File: purchase_work_acceptance/models/account_move.py [hunk 1]
```diff
@@ -22,9 +22,11 @@ class AccountMove(models.Model):
...
 
     def _compute_require_wa(self):
-        self.require_wa = self.env.user.has_group(
-            "purchase_work_acceptance.group_enforce_wa_on_invoice"
-        )
+        self.require_wa = False
+        if self.type == "in_invoice":
+            self.require_wa = self.env.user.has_group(
+                "purchase_work_acceptance.group_enforce_wa_on_invoice"
+            )
```
Comment (kittiu) [inline purchase_work_acceptance/models/account_move.py]:
```suggestion
        for rec in self:
            enforce_wa = self.env.user.has_group(
                "purchase_work_acceptance.group_enforce_wa_on_invoice"
            )
            rec.require_wa = rec.type == "in_invoice" and enforce_wa
```

**Review by Saran440 — APPROVED** [review]
LGTM :+1:

**Review by ps-tubtim — APPROVED** [review]

**Comment by kittiu** [comment]
cc @etobella not sure you want to give a comment.

**Comment by Saran440** [comment]
Updated [DONE, Ready to review and Merge]
- Fix validate error button set to draft  in WA when WA related with Receipts
- Fix onchange quantity (value not updated)
- Fix invoice view not display `wa_id` when you go to invoicing > vendors > bills
- Improved test script

**Comment by HviorForgeFlow** [comment]
@kittiu is ready for merge?

**Comment by kittiu** [comment]
@HviorForgeFlow yes thank you

**Pull request merged**